src/components/Layout/PhoneNumberInput.tsx

```diff
@@ -2,6 +2,9 @@ import React, { FunctionComponent } from "react";
 import { View, TextInput, StyleSheet } from "react-native";
 import { AppText } from "./AppText";
 import { size, color, borderRadius, fontSize } from "../../common/styles";
+import { PhoneNumberFormat, PhoneNumberUtil } from "google-libphonenumber";
+
+const MAX_LENGTH_FOR_NSN = 17;
```

Comment:
While this value is strictly from the library **now**, what if it changes in the future? How can we implement this in a better way? 

Can we consider leveraging the library in some way to do the _heavy lifting_?

Reply:
yes, MAX_LENGTH_FOR_NSN = 17 removed and changed to phoneNumberFormatted function

Make code cleaner with reuseable function phoneNumberFormatted and try catch to leverage on errors format and parse based on google-libphonenumber

Reply:
Thanks!